Library/Homebrew/github_packages.rb

```diff
@@ -225,21 +240,25 @@ def upload_bottle(user, token, skopeo, formula_name, bottle_hash)
         digest:      "sha256:#{manifest_json_sha256}",
         size:        manifest_json_size,
         platform:    platform_hash,
-        annotations: {},
+        annotations: { "org.opencontainers.image.ref.name" => tag },
       }
     end
+    formula_annotations_hash["org.opencontainers.image.created"] = created_times.max.strftime("%F")
 
-    index_json_sha256, index_json_size = write_image_index(manifests, blobs)
+    index_json_sha256, index_json_size = write_image_index(manifests, blobs, formula_annotations_hash)
 
     write_index_json(index_json_sha256, index_json_size, root)
 
-    image = "#{URL_DOMAIN}/#{org}/#{repo}/#{formula_name}"
+    image = "#{DOCKER_PREFIX}/#{org}/#{repo}/#{formula_name}"
     image_tag = "#{image}:#{version_rebuild}"
     puts
-    system_command!(skopeo, verbose: true, print_stdout: true, args: [
-      "copy", "--dest-creds=#{user}:#{token}",
-      "oci:#{root}", "docker://#{image_tag}"
-    ])
+    args = ["copy", "--all", "oci:#{root}", image_tag.to_s]
+    if dry_run
```

Comment:
This doesn't really feel like a dry run given it's written and removed a bunch of files above. I'd suggest we output that JSON instead of writing it to disk if desired.

Reply:
I want to output the complete OCI image directory, but not upload it to the remote host. That could be done with a separate `--write-oci` flag if desired.